Templates/local_openshift/hogarama-fluentd.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
```

Comment:
this will be autogenerated when creating the imagestream and does not need to be in the export which results in more readable/cleaner yaml file.